src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -2179,6 +2221,22 @@ def plant_overhead_cost_rule(c, t):
                     + c.additional_waste_cost / pyunits.year
                 )
 
+        # sum of variable operating costs of membrane units
```

Comment:
Is this specific to membrane units in WaterTAP, or general for any WaterTAP unit model?

Reply:
General for any WaterTAP unit model has a costing method called by the `WaterTAPCosting` or `ZeroOrderCosting` classes.

Reply:
Okay, the comment in L2224 could be reworded to not be exclusive to membranes. Either way, the code itself directly below the comment is clear.